platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## master
+
+* Fix symbol querying for annotations near tile boundaries at high zoom. ([#12472](https://github.com/mapbox/mapbox-gl-native/issues/12472))
```

Comment:
Apparently the iOS map SDK doesn’t publicly expose a dedicated annotation-querying method. However, this blurb could refer to both `-[MGLMapView visibleFeaturesAtPoint:]` and annotation selection via tapping.